website/docs/r/query.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "Query"
+layout: "lacework"
+page_title: "Lacework: lacework_query"
+description: |-
+  Create and manage Queries
+---
+
+# lacework\_query
+
+To provide customizable specification of datasources, Lacework provides the Lacework Query Language (LQL). 
+LQL is a human-readable text syntax for specifying selection, filtering, and manipulation of data. 
+It overlaps with SQL in its syntax and what it allows.
+
+For more information, see the [LQL Overview Documentation](https://docs.lacework.com/lql-overview).
+
+## Example Usage
```

Comment:
I wish we could add at least two examples, one that doesn't have evaluator ID and another one that NEEDS this evaluator ID and explain why the user needs it.